backend/src/main/java/io/github/pietroow/real_estate_monitoring/controller/ClienteController.java

```diff
@@ -0,0 +1,50 @@
+package io.github.pietroow.real_estate_monitoring.controller;
+
+import io.github.pietroow.real_estate_monitoring.dto.ClienteRequestDTO;
+import io.github.pietroow.real_estate_monitoring.dto.ClienteResponseDTO;
+import io.github.pietroow.real_estate_monitoring.mapper.ClienteMapper;
+import io.github.pietroow.real_estate_monitoring.model.Cliente;
+import io.github.pietroow.real_estate_monitoring.service.ClienteService;
+import jakarta.validation.Valid;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/clientes")
+public class ClienteController {
+
+    private final ClienteService clienteService;
+
+    public ClienteController(ClienteService clienteService) {
+        this.clienteService = clienteService;
+    }
+
+    @PostMapping
+    public ClienteResponseDTO criar(@RequestBody @Valid ClienteRequestDTO dto) {
+        Cliente cliente = ClienteMapper.toEntity(dto);
+        Cliente salvo = clienteService.salvar(cliente);
```

Comment:
```suggestion
        Cliente cliente = ClienteMapper.toEntity(dto);
```

@SanteLGG dar um auxilio pro nikolas implementar o mapper da forma correta